### DISCUSSION TITLE
Gargulec podróżuje

### DISCUSSION THREAD

**Discussion by hawkeye116477** [MajkiIT/polish-ads-filter#17275] [Q&A]
Słyszałem, że @gargul03 to ponoć wielki podróżnik, któremu nic niestraszne. Z jakiś tydzień temu był w USA w sprawie Huaweia, a teraz jest w Korei i zwiedza siedzibę Samsunga, a jeszcze poprzednio był nawet w Chinach i odwiedził siedzibę Huaweia, a po drodze wstąpił też do Chile po polecone mu papryczki, które jak się potem okazało były zbyt ostre, więc pojechał do ruskich internautów, by im osobiście odpowiednio podziękować (ponoć wykonał jakieś gestykulacje w ich stronę). Czy to prawda?

**Comment by xxcriticxx** [comment]
wtf?
